pylint/checkers/spelling.py

```diff
@@ -417,10 +417,17 @@ def _check_spelling(self, msgid: str, line: str, line_num: int) -> None:
                 col += word_start_at
                 if starts_with_comment:
                     col += 1
-                indicator = (" " * col) + ("^" * len(word))
                 all_suggestion = "' or '".join(suggestions)
-                args = (word, original_line, indicator, f"'{all_suggestion}'")
-                self.add_message(msgid, line=line_num, args=args)
+                self.add_message(
+                    msgid,
+                    line=line_num,
+                    args=(
+                        word,
+                        original_line,
+                        (" " * col) + ("^" * len(word)),
+                        f"'{all_suggestion}'",
+                    ),
+                )
```

Comment:
Here we go from 2 lines to 7 just for the sake of following a metric. Imo the code is worse. There's also the very real possibility of introducing bug while doing it. Let's not refactor for refactoring sake.